src/main/java/se/sundsvall/incident/service/mapper/Mapper.java

```diff
@@ -1,97 +1,126 @@
 package se.sundsvall.incident.service.mapper;
 
+import java.util.List;
+import java.util.UUID;
+
 import org.springframework.stereotype.Component;
+
 import se.sundsvall.incident.api.model.AttachmentRequest;
+import se.sundsvall.incident.api.model.CategoryDTO;
+import se.sundsvall.incident.api.model.CategoryPost;
 import se.sundsvall.incident.api.model.IncidentSaveRequest;
 import se.sundsvall.incident.dto.AttachmentDto;
 import se.sundsvall.incident.dto.Category;
 import se.sundsvall.incident.dto.IncidentDto;
 import se.sundsvall.incident.dto.Status;
 import se.sundsvall.incident.integration.db.entity.AttachmentEntity;
+import se.sundsvall.incident.integration.db.entity.CategoryEntity;
 import se.sundsvall.incident.integration.db.entity.IncidentEntity;
 
-import java.util.List;
-import java.util.UUID;
-
 @Component
```

Comment:
Why is this class a Spring bean? This could be a non-instantiable class with static methods instead.